lib/smart_answer/flow.rb

```diff
@@ -16,18 +16,18 @@ def initialize(&block)
       instance_eval(&block) if block_given?
     end
 
+    def append(flow)
+      flow.nodes.each do |node|
+        node.flow = self
```

Comment:
I wondered whether this would play nicely with the `FlowRegistry` caching that happens in production. I enabled `preload_flows` in my local version of this branch and accessed both am-i-getting-minimum-wage and minimum-wage-calculator-employers and they both appear to work as expected. Note that I didn't run the server in production mode as I was having problems connecting to the asset server in integration.